specification/paloaltonetworks/resource-manager/PaloAltoNetworks.Cloudngfw/preview/2025-07-07-preview/PaloAltoNetworks.Cloudngfw.json

```diff
@@ -7744,6 +7829,7 @@
       "in": "path",
       "required": true,
       "description": "Firewall resource name",
+      "pattern": "^[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?$",
```

Comment:
Breaking change is happening due to this pattern on firewall resource name. This was required to fix a lint check which mentioned that this name should follow a pattern. 

This pattern indicates that name can contain alphanumeric characters along with a hyphen and hyphen should not occur at the beginning and end of the name. 

This check is already there in code.

<img width="725" height="119" alt="Screenshot 2025-07-17 180510" src="https://github.com/user-attachments/assets/89a244d6-7896-45cf-a124-221d9df63ac7" />